website/content/en/status/report-2024-01-2024-03/openstack.adoc

```diff
@@ -0,0 +1,24 @@
+=== OpenStack on FreeBSD
+
+Links: +
+link:https://www.openstack.org/[OpenStack] URL: link:https://www.openstack.org/[] +
+link:https://github.com/openstack-on-freebsd[OpenStack on FreeBSD] URL: link:https://github.com/openstack-on-freebsd[]
+
+Contact: Chih-Hsin Chang <starbops@hey.com> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+The OpenStack on FreeBSD project aims to seamlessly integrate OpenStack cloud infrastructure with the FreeBSD operating system.
+It uses FreeBSD's unique features while ensuring compatibility with OpenStack standards.
+
+In the first quarter of 2024, we made significant progress on the OpenStack on FreeBSD project.
+This included submitting a proposal for BSDCan 2024 and attending AsiaBSDCon 2024 to link:https://papers.freebsd.org/2024/asiabsdcon/chang-towards-a-robust-freebsd-based-cloud-porting-openstack-components/[share our porting experiences and gain exposure for the project].
+The feedback received at AsiaBSDCon was particularly valuable and helped in refining the project's direction.
+During this period, we also reviewed the project's phase 1 tasks and made necessary adjustments.
+We also planned for phases 2 and 3, aligning them with the project's long-term goals.
+One technical achievement was verifying the functionality of bhyve serial console over TCP, an important part of the project's infrastructure.
+Additionally, we created a link:https://asciinema.org/a/647308[demo video] showcasing the project's progress and features.
+
+Looking ahead, our focus for the next quarter includes confirming the feasibility of implementing FreeBSD privilege management user space tool leveraging man:mac[4] and man:priv[9], simplifying installation steps by transitioning to FreeBSD ports, and porting OpenStack Ironic to FreeBSD.
```

Comment:
Should be either "... tools" or "a .... tool".

Also, s/privilege management/privilege-management/, I think.

Reply:
Updated, thank you.